lib/ReactViews/DataCatalog/DataCatalogGroup.jsx

```diff
@@ -88,6 +88,18 @@ const DataCatalogGroup = observer(
       return url.replace(/^https?:\/\//, "");
     },
 
+    componentDidMount() {
+      if (this.props.group.isOpen) {
+        this.props.group.loadMembers();
+      }
+    },
```

Comment:
On mount/update ensure that the group is loaded if it is open, this is so that `GroupMixin.refreshKnownContainerIds()` method is run so that the items ancestors are set correctly.